SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+If you think you have found a vulnerability in TiDB products or have experienced security incidents involving vulnerabilities in TiDB products, please report this vulnerability as soon as possible to help us solve it.
+We recommend that you provide as much vulnerability information as possible in the following format and send an email to the TiDB security team mailbox (security@tidb.io):
+issue title*:
+Overview*:
+Affected components and version number*:
+CVE number (if any):
+Vulnerability verification process*:
+contact details*:
+*Indicates required items.
+
+prompt:
+TiDB takes all vulnerabilities very seriously. We will confirm the vulnerabilities you submit within 48 hours (working days) and keep in touch with you.
+When a security vulnerability fix announcement is issued, TiDB will thank and praise the reporter. At the same time, in order to gain credibility, security researchers must follow a responsible approach, including:
```

Comment:
```suggestion
When a security vulnerability fix announcement is issued, TiDB will thank and praise the reporter. At the same time, in order to gain credibility, security researchers must follow a responsible approach, including:
```
Can we have another statement rather than 'thank and praise the report'? Do we have a detailed plan for the appreciation?

Reply:
Can we have another statement rather than 'thank and praise the report'? 
ok，updated
Do we have a detailed plan for the appreciation?
No，there is no such plan now.